recipes/dttlib/meta.yaml

```
@@ -0,0 +1,81 @@
+{% set name = "dttlib" %}
+{% set version = "0.3.4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://git.ligo.org/cds/software/cr_tools/dttlib/-/archive/{{ version }}/{{ version }}.tar.gz
+  sha256: 3071c249c45383a7bda9e3de9cfa0f8b185283dec3ffa6d2c43fb91f5f24b424
+
+build:
+  skip: true  # [win or osx]
```

```suggestion
  skip: true  # [win or osx]
```
Did you disable OSX and Windows because they were failing?  If so, can you please re-enable them so we can see the errors?  It would be better if we can see what is failing for them and fix the issues if possible.

---

I would also need to get some dependencies working on windows and OSX, such as libcds and gds-sigp.

---

I see, in that case skipping those OSes is fine for the moment but it would be a good idea to add OSX support at least to both the dependencies and this package.  Windows can obviously be much more difficult so it's usually less of a priority.